### DISCUSSION TITLE
What does nerf training speed have to do with?

### DISCUSSION THREAD

**Discussion by qhdqhd** [NVlabs/instant-ngp#664] [Q&A] [answered]
What does nerf training speed have to do with? My experiment found that it has a lot to do with the resolution of the image, and it may also be related to the number of images and the complexity of the scene?
My dataset can not finish training in 5 seconds, maybe 1 minutes.

**Comment by qhdqhd** [comment]
hello? Anybody can help?

**Reply by ENXING** [reply]
The fox should be trained with serveral seconds in RTX 3080. My own scene will converge within serveral minites. Maybe it is normal.

**Comment by Tom94** [comment, marked as answer]
I would say it mostly has to do with the amount of detail / size of the scene.

The number of images and the resolution don't have an impact on performance (number of iterations per second), but larger, more complicated scenes need more iterations to converge to something that looks reasonable.
